src/content/docs/logs/logs-everywhere/ruby-telemetry-context.mdx

```diff
@@ -0,0 +1,45 @@
+
+* Cut through the noise of thousands of logs when troubleshooting time-critical issues, so you automatically see only the most relevant logs.
+* Navigate within multiple types of telemetry data, and have the data correlate back to the original issue.
+* Easily drill down into more detailed information from the same place in the UI.
+* Find the log lines that you need to identify and resolve a problem.
+
+![Telemetry in context in APM UI](./images/ruby-errors-log-details.png "Telemetry in context in APM UI")
+
+<figcaption>
+  **Telemetry in context:** From the left nav in the APM UI, you can select an error and immediately see related logs, log details, host name, important attributes, and more.
+</figcaption>
+
+## Install and configure [#enable]
+
+To enable telemetry in context for APM apps monitored by Ruby, all you need to do is:
+
+1. Install or update to [Ruby agent version 8.6.0 or higher](https://docs.newrelic.com/docs/release-notes/agent-release-notes/ruby-release-notes). **REVIEWER: need to confirm version**
+2. To collect other metadata, or to change default settings, update your [`newrelic.yml` configuration file](https://docs.newrelic.com/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration/). **REVIEWER: what optional configuration might they want to do?**
+
+When you update to telemetry in context, two changes are made to your configuration file:
+
+* Application log forwarding enabled by default
+* Maximum log sample stored (approximately 10,000 lines per minute by default)
+
+In addition, log decoration by default now includes `trace.id`, `span.id`, `entity.guid`, and `hostname`.
+
+**REVIEWER: Anything to add about what to do if they don't see telemetry data? The original Ruby logs in context doc has a troubleshooting section. https://docs.newrelic.com/docs/logs/logs-context/configure-logs-context-ruby#troubleshooting**
+
+## Explore your data [#view-ui]
+
+To make the most of your logging data:
+
+**REVIEWER: These bullets come from the original Ruby logs in context doc. Will need to be updated with more info about what they now see from APM and infrastructure.**
+
+* Set up [alerts](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/).
+* [Query your data](/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data/) and [create dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/).
+* Explore more logging data across your platform with our [Logs UI](/docs/logs/log-management/ui-data/use-logs-ui/). For example, with telemetry in context, you can easily see [log patterns](/docs/logs/ui-data/find-unusual-logs-log-patterns), or show [surrounding logs](/docs/logs/troubleshooting/find-issues-cause-or-impact-surrounding-logs) at the time an error occurred in your app.
```

Comment:
Other bullet ideas: 
* Use Errors Inbox to view the logs associated with your errors
* Visit Distributed Tracing to see the logs associated with individual traces
* Click on your Web Transaction Times chart on the APM Summary page to view logs associated with a specific point in time

(You likely know the best resources to link these pages to!)